<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2800</width>
    <height>1500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Webtomator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_2222">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridMain" rowstretch="0,0,0,0,0,0,0,0,0,0">
      <item row="6" column="0" colspan="5">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnSaveStory">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Save the shown story to your file system</string>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+S</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="4">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="4">
       <widget class="QPlainTextEdit" name="logList">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>3</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>400</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Courier</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="documentTitle">
         <string>Action Log</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Log</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="5">
       <widget class="Line" name="line_1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="btnLoadStory">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Load a story from your file system</string>
        </property>
        <property name="toolTipDuration">
         <number>-1</number>
        </property>
        <property name="whatsThis">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Load</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnPlay">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Run your story from start to end</string>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Run</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+R</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="4">
       <widget class="QTableWidget" name="storyStatementList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>4</verstretch>
         </sizepolicy>
        </property>
        <property name="baseSize">
         <size>
          <width>3</width>
          <height>3</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="iconSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <property name="cornerButtonEnabled">
         <bool>true</bool>
        </property>
        <property name="rowCount">
         <number>8</number>
        </property>
        <property name="columnCount">
         <number>4</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>200</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>50</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>50</number>
        </attribute>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <column>
         <property name="text">
          <string>Action</string>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Property</string>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Value</string>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Result</string>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <item row="0" column="3">
         <property name="text">
          <string/>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QComboBox" name="comboLogLevel">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Choose the minimum log level</string>
        </property>
        <item>
         <property name="text">
          <string>Debug</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Info</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Warning</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Error</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Critical</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="lblLogList">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Action Log</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QPushButton" name="btnClearLog">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Clear log text</string>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblStoryTitle">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>.SF NS Text</family>
        <pointsize>21</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#272727;&quot;&gt;Loaded Story:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWebtomator">
    <property name="title">
     <string>Webtomator</string>
    </property>
    <addaction name="actionLoadStory"/>
   </widget>
   <addaction name="menuWebtomator"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoadStory">
   <property name="text">
    <string>Load Story...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionLoadStory</sender>
   <signal>triggered()</signal>
   <receiver>btnLoadStory</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>false</bool>
  </property>
 </designerdata>
</ui>
